triton-vm/src/table/challenges.rs

```diff
@@ -11,6 +15,34 @@ use super::processor_table::ProcessorTableChallenges;
 use super::program_table::ProgramTableChallenges;
 use super::ram_table::RamTableChallenges;
 
+pub trait TableChallenges: Clone + Debug {
```

Comment:
This trait name is too generic to be helpful. Let's find a name that describes the features that are declared.